docs/source/nbclassic.rst

```diff
@@ -60,19 +60,37 @@ NbClassic in the Jupyter Ecosystem
 Read more details about the changes currently taking place in the
 Jupyter Ecosystem in this `team-compass issue`_.
 
-You can install the nbclassic, notebook 7 and jupyterlab, all three of
+**NbClassic and Notebook 7**
+
+You can install NbClassic, Notebook 7 and JupyterLab, all three of
 which will be providing different user interfaces that will be available
-on the same server.
+on the same server. When Notebook 7 is available, the NbClassic UI, will
+be served at the ``/nbclassic/tree`` base URL rather than the 
+base URL ``/tree`` used otherwise. If you are using Notebook 7 along 
+with NbClassic, you wil also have JupyterLab installed as it is a 
+dependency of Notebook 7, and these front ends will be available 
+through the following base URLs: JupyterLab at ``/lab``, Notebook 7 at 
+``/tree``, and NbClassic at ``/nbclassic/tree``.
+
+**NbClassic and Notebook 6**
 
```

Comment:
Would it be possible to add the sections
- NbClassic and JupyterLab 3.x: In that case, NbClassic does not have to be explicitly installed as JupyterLab 3.x depends on it. They will run on the same server, and are reachable via `/tree` for NbClassic and `/lab` for JupyterLab
- NbClassic and JupyterLab 4.x: In that case, NbClassic has to be installed explictelly. They will run on the same server, and are reachable via `/tree` for NbClassic and `/lab` for JupyterLab

It would be good also to split the section `NbClassic and Notebook 6` in. 2 (Notebooks 6.5.x and Notebooks <= 6.4.x).

Thx.

Reply:
Thank you for your review! I have implemented the changes you outlined! I think these additional sections are a must, I realize now that I missed documenting the scenario where Notebook is not installed.

Reply:
Thx @RRosio. I let you add the scenario where only NbClassic is installed before closing this comment.

Reply:
Thank you @echarles! I have added a section for a solo NbClassic install in the [NbClassic Independently section](https://github.com/jupyter/nbclassic/pull/196/files#diff-ed71f888851dd91aa432423b1c7ffc6e8ae62b0a7414308d26d3bfc758a67841R110-R120). I'm not sure if the mention of nbclassic enabled through Jupyter Server is quite necessary, but I thought I might add it any way and see if it fits in okay with the rest of that general section. If you'd suggest modifications I will be sure to update them!

Reply:
Updates sound good to me. I will resolve this comment. Thx again.